src/components/AdminOverlay/AdminOverlay.tsx

```diff
@@ -0,0 +1,307 @@
+    setCustomNodeUrl(e.target.value)
+  }
+
+  const handleSaveCustomNodeUrlClick = () => {
+    setNodeOverride(customNodeUrl)
+  }
+
+  return (
+    <>
+      <FormField title="Chain" dense>
+        <Select
+          items={environmentsItems}
+          onChange={handleEnvironmentChange}
+          value={targetDevEnv}
+          helperText={null}
+          disabled={BUILD_ENV === 'production'}
+        />
+      </FormField>
+
+      <FormField title="Node" dense>
+        <Checkbox label="Custom node URL" value={usingCustomNodeUrl} onChange={handleCustomNodeCheckboxChange} />
+        {!usingCustomNodeUrl ? (
+          <Select items={availableNodes} onChange={handleNodeChange} value={determinedNode} helperText={null} />
+        ) : (
+          <CustomNodeUrlWrapper>
+            <TextField value={customNodeUrl} onChange={handleCustomNodeUrlChange} helperText={null} />
+            <Button onClick={handleSaveCustomNodeUrlClick} size="large">
+              Save
+            </Button>
+          </CustomNodeUrlWrapper>
+        )}
+      </FormField>
+    </>
+  )
+}
+
+const StateTab: React.FC = () => {
+  const { displaySnackbar } = useSnackbar()
+
+  const handleExportClick = () => {
+    const storageKeys = Object.keys(window.localStorage)
+    const storage = storageKeys.reduce((acc, key) => {
+      const rawValue = window.localStorage.getItem(key)
+      if (rawValue) {
+        try {
+          acc[key] = JSON.parse(rawValue)
+        } catch (e) {
+          acc[key] = rawValue
+        }
+      }
+
+      return acc
+    }, {} as Record<string, unknown>)
+    const jsonStorage = JSON.stringify(storage)
+
+    const linkElement = document.createElement('a')
+    linkElement.setAttribute('href', 'data:application/json;charset=utf-8,' + encodeURIComponent(jsonStorage))
+    linkElement.setAttribute('download', `atlas-export-${new Date().toISOString()}.json`)
+    linkElement.click()
+  }
+
+  const handleFileChange = async (e: Event) => {
+    try {
+      const file = (e.target as HTMLInputElement).files?.[0]
+      if (!file) return
+      const fileText = await file.text()
+      const storage = JSON.parse(fileText)
+      Object.keys(storage).forEach((key) => {
+        window.localStorage.setItem(key, JSON.stringify(storage[key]))
+      })
+      displaySnackbar({
+        title: 'Local state updated',
+        iconType: 'success',
+      })
+    } catch (error) {
+      SentryLogger.error('Failed to import local state', 'AdminOverlay', error)
+      displaySnackbar({
+        title: 'JSON file seems to be corrupted',
+        description: 'Please try again with different file',
+        iconType: 'error',
+      })
+    }
+  }
+
+  const handleImportClick = () => {
+    const inputElement = document.createElement('input')
+    inputElement.setAttribute('type', 'file')
+    inputElement.onchange = handleFileChange
+    inputElement.click()
+  }
+
+  return (
+    <VerticalSpacedContainer>
```

Comment:
I know that's only for developers, but It would be nice to add some margin above these buttons. 
![image](https://user-images.githubusercontent.com/51168865/132565019-ced391d7-a50e-4dd3-8c60-b92bdc0b3169.png)